wiki/plugins/metadata/forms.py

```diff
@@ -82,6 +82,13 @@ def newArticle_ArticleCategory(self, name=None, parent=None, slug=None):
         newcategory.save()
         return newarticle, newcategory
 
+    def newArticle_without_category(self, name=None, parent=None, slug=None):
```

Comment:
OK, now `Adposition`, `Construal`, `Usage`, and `Corpus` instances will not have an associated category.